demo/src/index.js

```diff
@@ -5,12 +5,15 @@ import {
   Button, Grid, Header, Segment
 } from 'semantic-ui-react';
 
+import { Clause, Template } from '@accordproject/cicero-core';
 import { SlateTransformer } from '@accordproject/markdown-slate';
 
 import { render } from 'react-dom';
 import 'semantic-ui-css/semantic.min.css';
 import ContractEditor from '../../src/ContractEditor';
 
+const templates = {};
```

Comment:
This variable becomes global rather than local to the Clause plugin. Is that the intent?

Reply:
I forgot to remove this, just did now.